projects/valtimo/zgw/src/lib/modules/index.ts

```diff
@@ -18,4 +18,5 @@ export * from './documenten-api';
 export * from './objecten-api';
 export * from './contactmomenten';
 export * from './haalcentraal-brp';
+export * from './zaakdetails';
```

Comment:
Don't create new zaakdetails module; move to objecten-api module

Reply:
Its nice to keep it the same with the BE and there we also have a new module with the same name `zaakdetails`